.github/workflows/chromatic.yml

```diff
@@ -1,9 +1,6 @@
 name: Chromatic
 
-on:
-  push:
-    branches-ignore:
-      - 'changeset-release/**'
+on: push
```

Comment:
I don't think there is actually any reason to skip this - we can and probably should just run chromatic on changeset-release branches